stellargraph/data/explorer.py

```diff
@@ -1021,93 +756,137 @@ def _check_parameter_values(self, nodes, n, n_size, graph_schema, seed):
             graph_schema: <GraphSchema> None or a stellargraph graph schema object
             seed: <int> Random number generator seed; default is None
 
+        Returns:
+            The random state as determined by the seed.
         """
-        if nodes is None:
-            raise ValueError(
-                "({}) A list of root node IDs was not provided (nodes parameter is None).".format(
-                    type(self).__name__
-                )
-            )
-        if not is_real_iterable(nodes):
-            raise ValueError(
-                "({}) The nodes parameter should be an iterable of node IDs.".format(
-                    type(self).__name__
-                )
-            )
-        if (
-            len(nodes) == 0
-        ):  # this is not an error but maybe a warning should be printed to inform the caller
-            print(
-                "WARNING: ({}) No root node IDs given. An empty list will be returned as a result.".format(
-                    type(self).__name__
-                )
-            )
+        self._check_sizes(n_size)
 
-        if type(n) != int:
-            raise ValueError(
-                "({}) The number of walks per root node, n, should be integer type.".format(
-                    type(self).__name__
-                )
+        if graph_schema is not None and type(graph_schema) is not GraphSchema:
```

Comment:
I think this is already checked by `GraphWalk.__init__`?

Reply:
Yes.